php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1568,8 +1577,18 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     function getDataEntryCompletionStatus(): string
     {
-        $data = NDB_BVL_Instrument::loadInstanceData($this);
-        return $data["Data_entry_completion_status"];
+        $db = \Database::singleton();
+        if ($this->jsonData) {
+            // If JSON instrument, get Data_entry_completion_status
+            // from flag table column
+            return $db->pselectOne(
```

Comment:
Why does this behave differently for JSON instruments and PHP instruments? We should be getting the status from the same place in either case.

Reply:
In the case of a json instrument, loadInstanceData loads the `Data` column from the flag table. Since we are migrating the Data_entry_completion_status field out of the Data column and into its own column, it will not be included in loadInstanceData.